ttax/ops.py

```diff
@@ -368,3 +368,27 @@ def _mul_by_scalar(tt, c):
     return TTMatrix(cores)
   else:
     return TT(cores)
+
+
+def norm(tt, differentiable=False):
+  """Frobenius norm of TT object
+
+  :type tt: `TT` or `TTMatrix`
+  :param tt: TT object (tensor or matrix)
+  :type differentiable: bool
+  :param differentiable: whether to use a differentiable implementation or a fast implementation based on QR decomposition
+  :return: non-negative number which is the Frobenius norm of `tt`
+  :rtype: `float`
+  """
+
+  from ttax.decompositions import orthogonalize
+  if differentiable:
+    tt_einsum = TTEinsum(
```

Comment:
I think this is equivalent to a (simpler) `return jnp.sqrt(ops.flat_inner(tt, tt))`